census/tftransformestimator/README.md

```diff
@@ -0,0 +1,348 @@
+                       --train-filebase train \
+                       --eval-filebase test \
+                       --train-steps $TRAIN_STEPS \
+                       --job-dir $OUTPUT_DIR
+
+```
+
+### Using gcloud local
+Run the code on your local machine using `gcloud`. This allows you to mock
+running it on the cloud:
+
+```
+export TRAIN_STEPS=1000
+DATE=`date '+%Y%m%d_%H%M%S'`
+export OUTPUT_DIR=census_$DATE
+rm -rf $OUTPUT_DIR
+```
+
+```
+gcloud ml-engine local train --module-name trainer.task \
+                             --package-path trainer \
+                             -- \
+                             --tft-working-dir $TFT_OUTPUT_DIR_LOCAL \
+                             --train-filebase train \
+                             --eval-filebase test \
+                             --train-steps $TRAIN_STEPS \
+                             --job-dir $OUTPUT_DIR
+
+```
+
+### Using Cloud ML Engine
+Run the code on Cloud ML Engine using `gcloud`. Note how `--job-dir` comes
+before `--` while training on the cloud and this is so that we can have
+different trial runs during Hyperparameter tuning.
+
+```
+DATE=`date '+%Y%m%d_%H%M%S'`
+export JOB_NAME=census_$DATE
+export GCS_JOB_DIR=gs://<my-bucket>/path/to/my/jobs/$JOB_NAME
+echo $GCS_JOB_DIR
+export TRAIN_STEPS=5000
+```
+
+```
+gcloud ml-engine jobs submit training $JOB_NAME \
+                                    --stream-logs \
+                                    --runtime-version 1.4 \
+                                    --job-dir $GCS_JOB_DIR \
+                                    --module-name trainer.task \
+                                    --package-path trainer \
+                                    --region $REGION \
+                                    --project $PROJECT \
+                                    -- \
+                                    --tft-working-dir $TFT_OUTPUT_DIR \
+                                    --train-filebase train \
+                                    --eval-filebase test \
+                                    --train-steps $TRAIN_STEPS
+
+```
+
+## Tensorboard
+Run the Tensorboard to inspect the details about the graph.
+
+```
+tensorboard --logdir=$GCS_JOB_DIR
+```
+
+## Accuracy and Output
+You should see the output for default number of training steps and approx accuracy close to `80%`.
+
+# Distributed Node Training
+Distributed node training uses [Distributed TensorFlow](https://www.tensorflow.org/deploy/distributed).
+The main change to make the distributed version work is usage of [TF_CONFIG](https://cloud.google.com/ml/reference/configuration-data-structures#tf_config_environment_variable)
+environment variable. The environment variable is generated using `gcloud` and parsed to create a
+[ClusterSpec](https://www.tensorflow.org/deploy/distributed#create_a_tftrainclusterspec_to_describe_the_cluster). See the [ScaleTier](https://cloud.google.com/ml/pricing#ml_training_units_by_scale_tier) for predefined tiers
+
```

Comment:
Please update the docs link to ScaleTier: https://cloud.google.com/ml-engine/docs/pricing#scale-tier

Reply:
Done